consensus/polybft/state_test.go

```diff
@@ -1,23 +1,13 @@
 package polybft
 
 import (
-	"bytes"
 	"fmt"
-	"math/big"
 	"os"
 	"path"
-	"sync"
 	"testing"
 	"time"
 
-	"github.com/0xPolygon/polygon-edge/consensus/polybft/contractsapi"
-	bls "github.com/0xPolygon/polygon-edge/consensus/polybft/signer"
-	"github.com/0xPolygon/polygon-edge/types"
 	"github.com/hashicorp/go-hclog"
-	"github.com/stretchr/testify/assert"
-	"github.com/stretchr/testify/require"
-	"github.com/umbracle/ethgo"
-	"github.com/umbracle/ethgo/abi"
 )
 
 func newTestState(t *testing.T) *State {
```

Comment:
You can move this function to `helpers_test.go` and remove the `state_test.go` completely.

Reply:
Fixed https://github.com/0xPolygon/polygon-edge/pull/1055/commits/744c97ae877890d400757a57cdfc3b71611ae49e